packages/tools/src/tools/segmentation/strategies/eraseCircle.ts

```diff
@@ -1,22 +1,13 @@
-import type { Types } from '@cornerstonejs/core';
+import BrushStrategy from './BrushStrategy';
+import { CIRCLE_STRATEGY } from './fillCircle';
+import initializeErase from './utils/initializeErase';
 
-import { fillInsideCircle } from './fillCircle';
-import { LabelmapToolOperationData } from '../../../types';
+const ERASE_CIRCLE_STRATEGY = new BrushStrategy(
+  'EraseCircle',
+  initializeErase,
+  ...CIRCLE_STRATEGY.initializers
+);
 
-type OperationData = LabelmapToolOperationData & {
-  points: any; // todo fix
-};
+const eraseInsideCircle = ERASE_CIRCLE_STRATEGY.strategyFunction;
 
-export function eraseInsideCircle(
-  enabledElement: Types.IEnabledElement,
-  operationData: OperationData
-): void {
-  // Take the arguments and set the segmentIndex to 0,
-  // Then use existing fillInsideCircle functionality.
-  const eraseOperationData = {
-    ...operationData,
-    segmentIndex: 0,
-  };
-
-  fillInsideCircle(enabledElement, eraseOperationData);
-}
+export { ERASE_CIRCLE_STRATEGY, eraseInsideCircle };
```

Comment:
In this file and also in `eraseSphere`, I noticed that you export the strategy instances (`ERASE_CIRCLE_STRATEGY` and the other one for sphere), but they are not used anywhere in the code. I was initially wondering about the difference between these two, and I discovered that `ERASE_CIRCLE_STRATEGY` is not used anywhere. Can we simply export the actual one?